.github/workflows/windows.yml

```diff
@@ -2,74 +2,115 @@ name: Windows
 
 on:
   push:
-    branches: [ dev ]
+    branches: [ dev, ci ]
   pull_request:
-    branches: [ dev ]
+    branches: [ dev, ci ]
 
 jobs:
   build:
-    runs-on: [self-hosted, windows, x64]
+    runs-on: windows-latest
+
     strategy:
       fail-fast: false
       matrix:
-        TARGET: [java, python2, python3, javascript, csharp, dart, go, php]
+        TARGET: [
+          cpp,
+          csharp,
+          dart,
+          go,
+          java,
+          javascript,
+          python2,
+          python3,
+          php,
```

Comment:
Have you also tried to set up Swift? In theory, it should work, see https://github.com/antlr/antlr4/pull/3767

Reply:
Sure.